scanpy/tests/test_plotting.py

```diff
@@ -1218,6 +1218,15 @@ def test_scatter_raw(tmp_path):
     assert "Error" in comp, "Plots should change depending on use_raw."
 
 
+def test_binary_scatter(image_comparer):
+    save_and_compare_images = image_comparer(ROOT, FIGS, tol=15)
+    blobs = sc.datasets.blobs()
+    blobs.obs['binary'] = np.tile([True, False], 320)
+    sc.pp.pca(blobs)
```

Comment:
In order to make tests faster, instead of needing to compute a PCA, could you directly provide the coordinates for the embedding?

Reply:
@ivirshup I don't quite get it, sorry. What should I exactly compare to pre-computed PCA?

Reply:
you could just define your data as as list-of-lists with 4 or so datapoints instead of the dataset.